config/vendor_libraries.cmake

```diff
@@ -501,37 +501,24 @@ macro( setupLIBQUO )
 
 endmacro()
 
-
 #------------------------------------------------------------------------------
 # Setup Caliper (https://github.com/LLNL/Caliper)
 #------------------------------------------------------------------------------
 macro( setupCaliper)
 
-  if( NOT TARGET caliper )
+  if( NOT TARGET CALIPER::caliper )
     message( STATUS "Looking for Caliper...")
-
-    find_package( caliper QUIET
-      HINTS $ENV{CALIPER_ROOT_DIR}/share/cmake
-      )
-
-    if(caliper_FOUND)
-      message(STATUS "Looking for Caliper...found")
+    find_package( Caliper QUIET
+      HINTS $ENV{CALIPER_ROOT_DIR}/share/cmake )
```

Comment:
Is this going to use the new FindCaliper.cmake, or is it going to use the caliper-config.cmake from the caliper installation? If the former, would it be a good idea to remove the HINTS?

Reply:
I need to look at this again.  I didn't realize that caliper published import targets (find_package has too many modes!).  Marking PR as `WIP` until I review it.

Reply:
The files `caliper.cmake` and `caliper-release.cmake` that are installed with Caliper are really old school CMake. That is unfortunate.  I think this PR make better use of generic cmake targets with detailed properties.  Maybe newer versions of caliper won't need my `FindCaliper.cmake` helper.

Let me test the build w/o the `HINTS` option.

Reply:
HINTS is not actually required now.  I've removed it.